packages/frontend/component/src/theme/global.css

```diff
@@ -266,14 +266,6 @@ affine-block-hub {
   }
 }
 
-button,
```

Comment:
Why to remove?

Reply:
![CleanShot 2024-03-04 at 09 56 48](https://github.com/toeverything/AFFiNE/assets/13579374/1fd6bc40-e409-49ba-88d0-75dd421e3ab0)
Because there is a hidden button that needs to be able to drag.
So i deleted this global rule. and found that no-drag is defined locally in most places, so this global rule does not affect the features.

Reply:
Maybe it is better to override for this button? I'm not sure whether `pointer-events: none` worked here. I remember that @pengx17  has added this to the `button`, but maybe not it's parent wrapper.

Reply:
This was added for buttons or alike that are in blocksuite. Please do not remove these rules.

Reply:
This may be reverted